integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -423,17 +415,6 @@ def export(self, **kwargs):
                 incomplete_count,
             )
 
-            LOGGER.info(generate_formatted_log(
```

Comment:
added during a previous investigation, noisy and limited value after previous fix.